dev/VSIX/Extension/Cpp/Common/fr-FR/VSPackage.fr-FR.resx

```diff
@@ -204,21 +204,12 @@
     <value>Modèle de projet pour la création d’une application de bureau basée sur la Bibliothèque d’interface utilisateur Windows (WinUI 3) ainsi qu’un package MSIX pour le chargement indépendant ou la distribution via le Microsoft Store.</value>
   </data>
   <data name="1033" xml:space="preserve">
-    <value>Composant Windows Runtime (WinUI 3 dans UWP)</value>
```

Comment:
>dans UWP) [](http://example.com/codeflow?start=46&length=9)

The changes to this file (and all the other non-`en-us` ones) won't be preserved since it comes from the localization service. On the other hand this is a translation error... I'll need to look into how to file a bug against localization so that it gets updated in the next run of the localization pipeline.